coalib/core/DependencyTracker.py

```diff
@@ -293,7 +293,7 @@ def check_circular_dependencies(self):
          ...
         coalib.core.CircularDependencyError.CircularDependencyError: ...
 
-        :raises coalib.core.CircularDependencyError.CircularDependencyError:
+        :raises CircularDependencyError:
```

Comment:
I don't think full qualification is necessary. Let's discuss full qualification for references in another issue^^